### DISCUSSION TITLE
Flameshot won't run on macOS Mojave...

### DISCUSSION THREAD

**Discussion by rin67630** [flameshot-org/flameshot#2972] [Q&A]
Does an older version work?
I am really sad to must pass with this great tool, that I enjoy so much on Linux.
It would be fantastic to get it working for older macOS versions too.
Regards

**Comment by mmahmoudian** [comment]
I don't think there is any a major issue with compiling it on an older macOS. I haven't tried, but in theory it should work. Also compiling Flameshot is super easy if you have the required  dependencies installed.

**Reply by rin67630** [reply]
Not sure if a new compile is enough. 
Apple has been quite disruptive with their versions between Mojave and Catalina, the APIs to grab the screen may have dramatically changed.

**Reply by mmahmoudian** [reply]
Fair point. In Flameshot we are using Qt. if the Qt we use is supporting this particular macOS, then it would work without any issue, but if Qt has also dropped support for it, then I'm afraid there is not much we can do about it. I'm not a Mac user and haven't used Mac since Apple stopped supporting by Macbook Pro late 2012, so I consider myself as a terrible person to response to mac-specific questions esp. when it is to this level of detail. Perhaps other macOS users can bring more insight/help to the table.
